tests/test_ext_commands.py

```diff
@@ -187,6 +187,19 @@ def test_sanitize_repo_return_fail_when_has_untracked_files(
     assert "untracked file" in result.msg
 
 
+def test_plug_error_raised_if_file_list_doesnt_exist(
```

Comment:
As a general note, write test case names in present tense. The name should read "this happens", rather than "this happened". I.e. here it should be `test_raises_plug_error_if_file_list_doesnt_exist`.

Another issue with this test case name is that it doesn't say what function it's testing, and there's no docstring saying that either. So a more complete name would be `test_sanitize_repo_raises_plug_error_if_file_list_doesnt_exist`. This is also why I usually namespace tests with classes, because then _what_ is being tested is implied by the namespace.